### ISSUE TITLE
OpenSprinkler Operation as a switch

### ISSUE THREAD

**Issue by etxttul** [vinteo/hass-opensprinkler#39]
Hi,

I have installed opensprinkler add-on. It seems to work well, thanks. 
I have a question, would it be possible to inplement binary_sensor.opensprinkler_operation as a switch?

The reason for this request is that the weather service included in opensprinkler is not that accurate.
I like to use home assistant to look in to my local weather service in HA and if it rains, I like to just disable the operation until it stops raining with automations in HA.

Best Regards
Tobe

**Comment by nagyrobi** [comment]
+1 for this.

My reason is that despite the weather service coming through opensprinkler states that temperature is under freezing point, scheduled watering is being run. 
I'd like to override opensprinkler operation from Home Assistant based on my outdoor temp sensor. As soon as temperature goes below 1C, disable opensprinkler operation, so by the time it reaches freezing temp, all waterings should stop if in progress, or not run.

**Comment by sdwilsh** [comment]
I think you'd want #41 for this to disable a program based on HA data.

**Comment by nagyrobi** [comment]
Not a certain program, but block the whole OpenSprinkler operation from HA. OpenSprinkler has a great weather-based (Evapotranspiration calculated) irrigation algorithm, re-programming it in HA is useless and hard to do. All thats needed is to be able to toggle the master switch of OpenSprinkler from HA.

**Comment by vinteo** [comment]
Feature available in version 0.2.0, please update via HACS. Please open another issue if there any are issues.

**Comment by nagyrobi** [comment]
Excellent! Thank you very much!

**Comment by etxttul** [comment]
Hi, It works perfectly, Thanks!!!!

Tobe

**Comment by nagyrobi** [comment]
Indeed. During this morning's freezes OpenSprinkler was blocked by HA as programmed by the automation. Cool!
